src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -251,6 +258,11 @@ private static PerLanguageOption<T> CreateOption<T>(OptionGroup group, string na
                 EditorConfigStorageLocation.ForBoolCodeStyleOption("dotnet_style_readonly_field"),
                 new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.PreferReadonly") });
 
+        internal static readonly Option<string> FileHeaderTemplate = CreateCommonOption(
```

Comment:
So we will have a single option that will be applicable for all languages and users cannot specify different values per-language? Or is this primarily due to the fact that this option will always be editorconfig based and hence the option itself will be agnostic but can be configured differently for each file extension type?

Reply:
> ... this option will always be editorconfig based and hence the option itself will be agnostic but can be configured differently for each file extension ...

This is exactly what happened.